common/types/mapper/thrift/matching.go

```diff
@@ -101,7 +101,7 @@ func FromCancelOutstandingPollRequest(t *types.CancelOutstandingPollRequest) *ma
 		DomainUUID:   &t.DomainUUID,
 		TaskListType: t.TaskListType,
 		TaskList:     FromTaskList(t.TaskList),
-		PollerID:     t.PollerID,
+		PollerID:     &t.PollerID,
```

Comment:
Same Q/assumption as the "forwarded from" field PR

Reply:
Yes, the `pollerID` is internal. It is set in the frontend with `uuid.New()`.